delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/disk/DiskStorageSaveFile.scala

```diff
@@ -24,17 +25,17 @@ import java.nio.file._
 import java.util.UUID
 import scala.concurrent.Future
 
-final class DiskStorageSaveFile(storage: DiskStorage)(implicit as: ActorSystem, uuidf: UUIDF) extends SaveFile {
+final class DiskStorageSaveFile(implicit as: ActorSystem, uuidf: UUIDF) {
```

Comment:
Any reason it is not included in `DiskFileOperations` ?

Reply:
Only that it's a bit more involved, happy to move that and `S3StorageSaveFile` to have it all in one place